.github/workflows/test-dbt-installation-main.yml

```diff
@@ -13,14 +13,29 @@ permissions:
 
 on:
   workflow_dispatch:
+    inputs:
+      test-installation-method:
+        description: "Choice installation method to test"
```

Comment:
```suggestion
        description: "Choose installation method(s) to test"
```

Reply:
Fixed in https://github.com/dbt-labs/actions/pull/68/commits/df4eb6faaedf3bad5d6354e0c464f4c7a5864b63